detections/application/cisco_secure_application_alerts.yml

```diff
@@ -0,0 +1,88 @@
+name: Cisco Secure Application Alerts
+id: 9982bff4-fc5d-49a3-ab9e-2dbbab2a711b
+version: 1
+date: '2025-02-04'
+author: Ryan Long, Bhavin Patel, Splunk
+status: production
+type: Anomaly
+description: |
+  The following analytic is to leverage alerts from Cisco SecureApp, which identifies and monitors exploit attempts targeting business applications. The primary attack observed involves exploiting vulnerabilities in web applications, including injection attacks (SQL, API abuse), deserialization vulnerabilities, remote code execution attempts, LOG4J and zero day attacks. These attacks are typically aimed at gaining unauthorized access, exfiltrating sensitive data, or disrupting application functionality.
+
+  Cisco SecureApp provides real-time detection of these threats by analyzing application-layer events and correlating attack behavior with known vulnerability signatures. This detection methodology helps the Security Operations Center (SOC) by:
+
+  * Identifying active exploitation attempts in real-time, allowing for quicker incident response.
+  * Categorizing attack severity to prioritize remediation efforts based on risk level.
+  * Providing visibility into attacker tactics, including source IP, attack techniques, and affected applications.
+  * Generating risk-based scoring and contextual alerts to enhance decision-making within SOC workflows.
+  * Helping analysts determine whether an attack was merely an attempt or if it successfully exploited a vulnerability.
+
+  By leveraging this information, SOC teams can proactively mitigate security threats, patch vulnerable applications, and enforce security controls to prevent further exploitation.
+data_source:
+- Cisco Secure Application AppDynamics Alerts
+search: |-
+  `appdynamics_security` blocked=false 
+  | rename attackEvents{}.attackOutcome AS attackOutcome, "attackEvents{}.vulnerabilityInfo.*" AS * 
+  | fields - tag::eventtype, eventtype, host, id, index, linecount, punct, source, sourcetype, splunk_server, tag, SourceType, app clientAddressType, application, tier, "attackEvents{}.*" 
```

Comment:
Is there a reason we're doing this?

Reply:
yep , i believe the 3.0.0 AppDynamic TA does have the parsing yet.

Reply:
I specifically mean removing lots of the built-in to splunk fields. Not sure why we're removing them.

Reply:
It _should_ be fine, just feels like something that will raise questions + take some additional time to grok for someone just looking at the search.